### PR TITLE
docs: update the docs for streaming messages

### PR DESCRIPTION
**Pull request by vbarda**

### CODE REVIEW

File: docs/docs/cloud/concepts/api.md [hunk 1]
```diff
@@ -62,122 +62,7 @@ The LangGraph Cloud API offers several features to support complex agent archite
...
-Streaming mode `events` is the same as using `.astream_events` in the LangGraph library - for a deeper conceptual explanation of this, you can see the LangGraph library documentation [here](../../concepts/low_level.md#streaming).
-
-#### `mode="messages"`
-Streaming mode `messages` is a new streaming mode, currently only available in the API. What does this mode enable?
-
-This mode is focused on streaming back messages. It currently assumes that you have a `messages` key in your graph that is a list of messages. Assuming we have a simple react agent deployed, what does this stream look like?
-
-All events emitted have two attributes:
-
-- `event`: This is the name of the event
```
Comment (eyurtsev) [inline docs/docs/cloud/concepts/api.md:88]: the format might be goodto keep

Comment (eyurtsev) [reply docs/docs/cloud/concepts/api.md:88]:
```python
StreamPart(event='messages/metadata', data={'run-985c0f14-9f43-40d4-a505-4637fc58e333': {'metadata': {'created_by': 'system', 'run_id': '1ef657de-7594-66df-8eb2-31518e4a1ee2', 'graph_id': 'agent', 'thread_id': 'c178eab5-e293-423c-8e7d-1d113ffe7cd9', 'model_name': 'openai', 'assistant_id': 'fe096781-5601-53d2-b2f6-0d3403f7e9ca', 'langgraph_step': 1, 'langgraph_node': 'agent', 'langgraph_triggers': ['start:agent'], 'langgraph_task_idx': 0, 'ls_provider': 'openai', 'ls_model_name': 'gpt-4o', 'ls_model_type': 'chat', 'ls_temperature': 0.0}}})
```

and explain how this is different from the library representation

File: docs/docs/cloud/concepts/api.md [hunk 2]
```diff
@@ -62,122 +62,7 @@ The LangGraph Cloud API offers several features to support complex agent archite
...
-- `messages`: Stream complete messages (at the end of node execution) as well as tokens for any messages generated inside a node. This mode is primarily meant for powering chat applications. This is only an option if your graph contains a `messages` key. See the [how-to guide](../how-tos/stream_messages.md) for streaming messages.
-- `updates`: Streams updates to the state of the graph after each node is executed. See the [how-to guide](../how-tos/stream_updates.md) for streaming updates.
-- `events`: Stream all events (including the state of the graph) that occur during graph execution. See the [how-to guide](../how-tos/stream_events.md) for streaming events. This can be used to do token-by-token streaming for LLMs.
-- `debug`: Stream debug events throughout graph execution. See the [how-to guide](../how-tos/stream_debug.md) for streaming debug events.
-
-You can also specify multiple streaming modes at the same time. See the [how-to guide](../how-tos/stream_multiple.md) for configuring multiple streaming modes at the same time.
-
-See the [API reference](../reference/api/api_ref.html#tag/runscreate/POST/threads/{thread_id}/runs/stream) for how to create streaming runs.
```
Comment (eyurtsev) [inline docs/docs/cloud/concepts/api.md:75]: API Reference might make sense to include as well

File: docs/docs/cloud/concepts/api.md (hunk 1 above)
Comment (vbarda) [reply docs/docs/cloud/concepts/api.md:88]: yea this could be useful, i will add back

Comment (vbarda) [reply docs/docs/cloud/concepts/api.md:88]: (will put this in the other doc)

**Pull request merged**